typespec-extension/src/main/java/com/azure/autorest/fluent/TypeSpecFluentPlugin.java

```diff
@@ -99,7 +99,7 @@ protected FluentMapper getFluentMapper() {
 
         SETTINGS_MAP.put("license-header", "MICROSOFT_MIT_SMALL_TYPESPEC");
 
-        SETTINGS_MAP.put("generic-response-type", true);
+        SETTINGS_MAP.put("generic-response-type", false);
```

Comment:
Haven't read the details yet, but yeah I think mgmt from swagger have it false.

Reply:
Yeah. This PR did:
1. Set generic-response-type to false for mpg, aligning with mgmt plane.
2. Fixed the scenario where generic-response-type is true.